hazelcast-spring/src/main/resources/hazelcast-spring-3.11.xsd

```diff
@@ -1213,6 +1213,60 @@
                                 </xs:attribute>
                             </xs:complexType>
                         </xs:element>
+                        <xs:element name="merkle-tree" minOccurs="0" maxOccurs="unbounded">
+                            <xs:complexType>
+                                <xs:annotation>
+                                    <xs:documentation>
+                                        Configuration for a merkle tree.
+                                        The merkle tree is a data structure used for efficient comparison of the
+                                        difference in the contents of large data structures. The precision of
+                                        such a comparison mechanism is defined by the depth of the merkle tree.
+                                        A larger depth means that a data synchronization mechanism will be able
+                                        to pinpoint a smaller subset of the data structure contents in which a
+                                        change occurred. This causes the synchronization mechanism to be more
+                                        efficient. On the other hand, a larger tree depth means the merkle tree
+                                        will consume more memory.
+                                        A smaller depth means the data synchronization mechanism will have to
+                                        transfer larger chunks of the data structure in which a possible change
+                                        happened. On the other hand, a shallower tree consumes less memory.
+                                        The depth must be between 2 and 27 (exclusive). The default depth is 10.
+                                        This configuration is not tied to a specific data structure and can be
+                                        reused.
+                                        See https://en.wikipedia.org/wiki/Merkle_tree.
```

Comment:
Also, it requires more turnarounds between the clusters to identify the leaves with different hashes. When setting the depth, the network latency also needs to be taken into account.